app/code/Magento/CustomerGraphQl/Model/Resolver/RequestPasswordResetEmail.php

```diff
@@ -0,0 +1,116 @@
+use Magento\Framework\Validator\EmailAddress as EmailValidator;
+
+/**
+ * Class Resolver for RequestPasswordResetEmail
+ */
+class RequestPasswordResetEmail implements ResolverInterface
+{
+    /**
+     * @var AuthenticationInterface
+     */
+    private $authentication;
+
+    /**
+     * @var CustomerRepositoryInterface
+     */
+    private $customerRepository;
+
+    /**
+     * @var AccountManagementInterface
+     */
+    private $customerAccountManagement;
+
+    /**
+     * @var EmailValidator
+     */
+    private $emailValidator;
+
+    /**
+     * RequestPasswordResetEmail constructor.
+     *
+     * @param AuthenticationInterface     $authentication
+     * @param CustomerRepositoryInterface $customerRepository
+     * @param AccountManagementInterface  $customerAccountManagement
+     * @param EmailValidator              $emailValidator
+     */
+    public function __construct(
+        AuthenticationInterface $authentication,
+        CustomerRepositoryInterface $customerRepository,
+        AccountManagementInterface $customerAccountManagement,
+        EmailValidator $emailValidator
+    ) {
+        $this->authentication = $authentication;
+        $this->customerRepository = $customerRepository;
+        $this->customerAccountManagement = $customerAccountManagement;
+        $this->emailValidator = $emailValidator;
+    }
+
+    /**
+     * Send password email request
+     *
+     * @param Field             $field
+     * @param ContextInterface  $context
+     * @param ResolveInfo       $info
+     * @param array|null        $value
+     * @param array|null        $args
+     *
+     * @return bool|Value|mixed
+     *
+     * @throws GraphQlInputException
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        if (empty($args['email'])) {
+            throw new GraphQlInputException(__('Email must be specified'));
+        }
+
+        if (!$this->emailValidator->isValid($args['email'])) {
+            throw new GraphQlInputException(__('Email is invalid'));
+        }
+
+        try {
+            $customer = $this->customerRepository->get($args['email']);
+        } catch (LocalizedException $e) {
+            throw new GraphQlInputException(__('Cannot reset customer password'), $e);
+        }
+
+        if (true === $this->authentication->isLocked($customer->getId())) {
+            throw new GraphQlInputException(__("The current customer isn't authorized"));
+        }
+
+        try {
+            return $this->customerAccountManagement->initiatePasswordReset(
+                $args['email'],
+                AccountManagement::EMAIL_RESET
+            );
+        } catch (LocalizedException $e) {
+            throw new GraphQlInputException(__("Cannot reset customer password"), $e);
```

Comment:
Please change to single quotes where possible.

Reply:
**Done**